spec/spec_helper.rb

```diff
@@ -3,9 +3,6 @@
 ( [2, 0, 0] <=> ( RUBY_VERSION.split(".").collect {|x| x.to_i} ) ) <= 0 or
   abort "Ruby v2.0.0 is required; this is v#{RUBY_VERSION}."
```

Comment:
This is the same snippet of code that got pruned out of the gemfile. Do we still need it here? I think it could maybe just be deleted since the Gemfile specifies the ruby version anyway. Are we looking at `RUBY_VERSION` anywhere else?

Reply:
Good catch.  This code is very WET (opposite of DRY).  I'll make a note to scan through the code for these manual version checks.